content/en/docs/tasks/job/pod-failure-policy.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Handling retriable and non-retriable pod failures with Pod failure policy
```

Comment:
These task contents are great! 😃

They really show the usefulness of the feature.